packages/amplify-provider-awscloudformation/lib/amplify-service-auth-check.js

```diff
@@ -0,0 +1,18 @@
+const { printAuthErrorMessage } = require('../src/aws-utils/aws-amplify');
+
+async function authDryRun(context, amplifyClient) {
```

Comment:
can we rename the function to `checkAmplifyServiceIAMPermission`

Reply:
you got it